core/src/main/java/org/radargun/stages/test/legacy/OperationLogic.java

```diff
@@ -7,28 +10,34 @@
  * @author Radim Vansa &lt;rvansa@redhat.com&gt;
  */
 public abstract class OperationLogic {
-   protected Stressor stressor;
+   protected LegacyStressor stressor;
 
    /**
     * Initialize this logic. No {@link org.radargun.Operation operations}
     * should be executed here.
     */
-   public void init(Stressor stressor) {
+   public void init(LegacyStressor stressor) {
       this.stressor = stressor;
    }
 
    /**
+    * Release resources held by this logic.
+    */
+   public void destroy() {}
```

Comment:
You must really hate providing extension points. Though, in this particular case, I have to admit that I haven't used that anywhere (usage got lost during refactoring).